en-US/M.xml

```diff
@@ -6,6 +6,16 @@
 <chapter id="m">
 	<title>M</title>
 	 <variablelist>
+		 <varlistentry id="macOS">
+		 <term>macOS</term>
+			<listitem>
+			 <para>
+				 In 2016, Apple rebranded OS X to macOS, adopting the nomenclature that it uses for their other operating systems, iOS, watchOS, and tvOS. The latest version of macOS is macOS 10.12 Sierra, which was publicly released in September 2016.	 
```

Comment:
I would introduce the list of "other operating systems" by using a colon. i.e., "...other operating systems: iOS, watchOS, and tvOS." Make that little patch and feel free to merge. Thanks.

Reply:
Thanks, David, appreciate the feedback. Added colon to the list.